### DISCUSSION TITLE
Use markdown and host on GitHub instead of Google Docs

### DISCUSSION THREAD

**Discussion by paulmillr** [FiloSottile/age#228] [Spec feedback]
Google Docs tracks users heavily. Why not use GitHub wiki, if you redirect age-encryption.org/ to GitHub anyway?

Also, a real textual format (markdown) would help a lot! Gdocs is a pain to read.

**Comment by paulmillr** [comment]
I've converted spec to markdown, feel free to add it to repo or github wiki https://gist.github.com/paulmillr/9c80bb176ee039272ab5c915d3c73afc

**Comment by PowerPress** [comment]
+1

**Comment by git001** [comment]
+1

**Comment by paulmillr** [comment]
Created pull request: #391

**Comment by FiloSottile** [comment]
The original spec was really more of a design document. I've extracted and polished the bits that made it into age, and published it at https://c2sp.org/age, where it will be maintained and versioned separately from the Go implementation.
